app/commands/utils.py

```diff
@@ -124,3 +126,100 @@ def get_user_locale(user_id, client):
     if user_locale["ok"] and (user_locale["user"]["locale"] in supported_locales):
         return user_locale["user"]["locale"]
     return default_locale
+
+
+def rearrange_by_datetime_ascending(text):
+    # Split the text by lines
+    lines = text.split("\n")
+
+    # Temporary storage for multiline entries
+    entries = []
+    current_entry = []
+
+    # Iterate over each line
+    for line in lines:
+        # Check if the line starts with a datetime format including 'EST'
+        if re.match(r"\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} EST", line):
```

Comment:
nit: this is super minor, but the timezone abbreviation changes between `EST` and `EDT` over the year.  Maybe just use something generic like `ET`?

Reply:
Good point Pat! I will make the change.

Reply:
Done!